gcp_variant_transforms/transforms/infer_headers_test.py

```diff
@@ -53,6 +53,17 @@ def _get_sample_header_fields(self):
         ('PS', Format('PS', 1, 'Integer', 'Special PS key'))])
     return vcf_header_io.VcfHeader(infos=infos, formats=formats)
 
+  def _get_sample_header_fields_with_annotation(self):
+    header = self._get_sample_header_fields()
+    header.infos['CSQ'] = OrderedDict([
```

Comment:
Please use `Info` and below here as well.

Reply:
Done. As discussed offline, I've modified `_get_sample_header_fields` so I can use `Info` here.